### DISCUSSION TITLE
Lab5 run_vivado.sh board part not found

### DISCUSSION THREAD

**Discussion by b3nsonchang** [bol-edu/HLS-SOC-Discussions#146] [Q&A] [answered]
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/86411659/44f94237-d2af-4761-8194-e099b2af20d4)
根據workbook 執行指令的時候，執行到source run_vivado.sh會發生以上error，想請問這個board part 要怎麼取得呢？

**Comment by ghost** [comment, marked as answer]
需要先設定板子資訊，你可以參考https://github.com/bol-edu/caravel-soc_fpga#prepare-a-xilinx-vitis-on-ubuntu-machin，上面有詳細步驟
